core/config/initializers/00_tenant_config_schema.rb

```diff
@@ -268,19 +273,22 @@
         # },
         organization_management: {
           type: "object",
+          title: "Organization management",
           properties: {
             enabled: {
               type: "boolean",
               default: true,
-              description: "Allow user to create and manage Organizations",
+              title: "Enabled",
+              description: "Allow your customer to create and manage multiple companies",
```

Comment:
This also disable:
- creation of new company
- the company settings menu on the left side